packages/server/database/types/Reflection.ts

```diff
@@ -56,7 +59,9 @@ export default class Reflection {
     this.plaintextContent = plaintextContent || extractTextFromDraftString(content)
     this.entities = entities
     this.isActive = true
+    this.isAnonymous = true
```

Comment:
stylistically, i'd probably use the nullish coalescing `this.isAnonymous = isAnonymous ?? true`, but it all does the same thing